plugins/modules/junos_package.py

```diff
@@ -13,7 +13,7 @@
 DOCUMENTATION = """
 module: junos_package
 author: Peter Sprygada (@privateip)
-short_description: Installs packages on remote devices running Junos
+short_description: (deprecated, removed after 2025-12-01) Installs packages on remote devices running Junos
```

Comment:
Are we really deprecating this one though? It should work without provider even if it doesn't technically use the connection plugin
```suggestion
short_description: (deprecated, removed after 2025-01-01) Installs packages on remote devices running Junos
```